platform/osgi/platform-osgi-configadmin/pom.xml

```diff
@@ -76,12 +91,24 @@
                             org.codice.felix.cm.file.Activator
                         </Bundle-Activator>
                         <Embed-Dependency>
-                            keyczar,
                             gson,
-                            org.osgi.core;inline=org/osgi/util/tracker/ServiceTracker*|org/osgi/util/tracker/AbstractTracked.class
+                            org.osgi.core;inline=org/osgi/util/tracker/ServiceTracker*|org/osgi/util/tracker/AbstractTracked.class,
+                            security-encryption-crypter,
+                            <!-- Crypter dependencies must be embedded in this bundle -->
+                            json,
+                            protobuf-java,
+                            tink
```

Comment:
❓ I believe you said something about needing these embeds, but the packages you need from `protobuf` and `tink` _should_ be coming it as package imports. Might just be necessary to explicitly add them to the `Import-Package` directive after the final exclude, if the `*` import isn't getting them.

Reply:
This bundle is super finicky when it comes to its imports. I tried importing the `tink`, `protobuf-java`, and `json` packages that I'm exporting from bundle 0, but this `platform-osgi-configadmin` bundle just won't resolve the imports for whatever reason.

I simplified it down to the point where I was exporting a single `tink` package from bundle 0, and importing it in both `platform-osgi-configadmin` and `encryption-service-impl`. The `encryption-service-impl` bundle was able to find the `tink` package perfectly; whereas the `platform-osgi-configadmin` bundle was still getting an `import package not resolved` error.

Reply:
Well then. It is a particularly hateful and hurtful bundle. Thanks for at least trying. 👍